ui/components/templates/components/AccountDropDown.jinja

```diff
@@ -0,0 +1,28 @@
+{# def #}
+
+<div class="ui dropdown">
```

Comment:
when we have a dedicated less component for this, we should use it (`ui account-dropdown dropdown`)